package-lock.json

```diff
@@ -47456,6 +47449,17 @@
         "@storybook/instrumenter": "^6.4.0",
         "@testing-library/jest-dom": "^5.16.2",
         "jest-mock": "^27.3.0"
+      },
+      "dependencies": {
```

Comment:
nit, probably nothing: that's a bit strange that this made it to the diff even though there isn't a change in package.json 🤔 

which node + npm version are you on?